RFC.rst

```diff
@@ -14,92 +14,82 @@ The goal of this code of conduct is to give a clear signal what is expected of c
 
 This proposal consists of several parts:
 
-  * A general "Code of Conduct", which provides a general overview on what constitutes as unacceptable behaviour
-  * A set of guidelines to foster collaboration on proposed language features through RFCs, in addition to the formal RFC process that we already have at https://wiki.php.net/rfc/voting
-  * A set of procedures to follow if individuals wish to make complaints if other project members do not follow either the Code of Conduct or the Constructive Collaboration Guidelines.
-  * A Contributor Mediation Team to assess and act upon the complaints as received through the above procedures
-  * Possibilities for strong action if things really go awry regarding a project member. For example, if project members continue to persistently ignore the Collaboration Guidelines, or violate the Code of Conduct, after several rounds of mediation and/or complaints.
+  * A general "Code of Conduct", which provides a general overview on what is expected and unacceptable behaviour.
```

Comment:
I would rephrase this as:
"on what is expected from project members and contributors and what constitutes unacceptable behaviour"
I think it's more readable, without changing the meaning.